community/pmc-tool/collector/main.py

```diff
@@ -59,25 +35,18 @@
 create_pull_requests_table(conn)
 
 # Retrieve list of pull requests
-authors_count = {}
-
+pull_requests = query_prs(API_ENDPOINT, headers)
```

Comment:
please create a new PR where we don't pass headers. The headers should be computed inside `gh.py`